syncode/runtime_eval.py

```diff
@@ -0,0 +1,40 @@
+from infer import Syncode
```

Comment:
what is this file?

Reply:
This file is for me to just run the runtime experiment automatically. I forgot to not include this in gitignore.